packages/gatsby-link/src/index.tsx

```diff
@@ -112,23 +145,23 @@ class GatsbyLink extends React.Component {
         style: { ...this.props.style, ...this.props.activeStyle },
       }
     }
-    return null
+    return {}
```

Comment:
need to look into this. This might have a functional difference..

Reply:
I used https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/reach__router/index.d.ts#L57 as a reference. The actual code: https://github.com/reach/router/blob/master/src/index.js#L408 simply spreads them. So think there should be no change